api/server/handlers/environment/list_deployments_by_cluster.go

```diff
@@ -147,6 +193,21 @@ func fetchOpenPullRequests(
 		return nil, err
 	}
 
+	var ghPRs []*github.PullRequest
+
+	for resp.NextPage != 0 && err != nil {
```

Comment:
```suggestion
	for resp.NextPage != 0 && err == nil {
```

I think this should be `err == nil`?